docs/prompting/few_shot/example_ordering.md

```diff
@@ -1,7 +1,33 @@
 ---
-title: ""
-description: ""
-keywords: ""
+title: "Example Ordering"
+description: "LLM outputs are heavily impacted by ordering of few shot examples"
 ---
 
-[wip]
+# Example Ordering
+
+The order of few-shot examples in the prompt can affect LLM outputs <sup><a href="https://arxiv.org/abs/2104.08786">1</a><a href="https://arxiv.org/abs/2106.01751">2</a><a href="https://arxiv.org/abs/2101.06804">3</a><a href="https://aclanthology.org/2022.naacl-main.191/">4</a></sup><sup><a href="https://arxiv.org/abs/2406.06608">\*</a></sup>. Consider permutating the order of these examples in your prompt to achieve better results.
+
+## Choosing Your Examples
+
+### Combinatorics
+
+One of the easiest methods is for us to manually iterate over each of the examples that we have and try all possible combinations we could create. This will in turn allow us to find the best combination that we can find.
+
```

Comment:
Maybe let's add one more method and we can merge this in? That way people can see different ways to combine examples